### DISCUSSION TITLE
Use hostinger's domain in vercel without affecting email

### DISCUSSION THREAD

**Discussion by pauldvlp** [vercel/community#1675] [Help]
### Summary

I want to use my domain registered with Hostinger, using the DNS (A Record), but without affecting the email, since changing the A Record will no longer send or receive emails to the addresses name@correo.com.

**Comment by swarnava** [comment]
Hi,

This should still be possible. What issue you are facing when you setup A record and point to Vercel?

**Comment by amyegan** [comment]
Email setup usually involves MX records rather than the A and CNAME records that are used for site setup. So, like Swarnava said, you should be able to change the A record without affecting your email.

Hostinger has a guide for email setup here: [support.hostinger.com/en/articles/4767973-how-to-set-up-a-domain-for-hostinger-email](https://support.hostinger.com/en/articles/4767973-how-to-set-up-a-domain-for-hostinger-email)

And GoDaddy has a nice explanation of how all of these records work which you can find here: [godaddy.com/garage/configuring-dns-for-email-a-quick-beginners-guide](https://www.godaddy.com/garage/configuring-dns-for-email-a-quick-beginners-guide/)
